donate/templates/fragments/donate_form.html

```diff
@@ -34,13 +34,17 @@
                         </label>
                     </div>
                     {% endfor %}
-
-                    <div class="donation-amount donation-amount--two-col donation-amount--other">
-                        <input type="radio" class="donation-amount__radio" name="amount" value="other" id="one-time-amount-other" autocomplete="off" data-other-amount-radio="">
-                        <label for="one-time-amount-other" class="donation-amount__label donation-amount-other__label" data-currency="">{% get_localized_currency_symbol initial_currency_info.code %}</label>
-                        <label for="one-time-amount-other-input" class="donation-amount__label--hidden">{% trans "Other amount" %}</label>
-                        <input type="number" class="donation-amount__input" id="one-time-amount-other-input" placeholder="{% trans 'Other amount' %}" min="{{ initial_currency_info.minAmount }}" max="10000000" data-other-amount="">
-                    </div>
+                    <div class="donation-amount--two-col">
```

Comment:
Wrapped both the input and the error message in a new div to achieve the effect requested from Kristina:

<img width="434" alt="image" src="https://user-images.githubusercontent.com/18314510/112698292-9abf1100-8e46-11eb-95d4-213977721211.png">